index.js

```diff
@@ -43,8 +47,8 @@ class ReadabilityView extends PureComponent {
 
       this.setState({
         cleanHtmlSource: !readabilityArticle
-          ? `<h1>Sorry, issue parsing ${url}</h1>`
-          : readabilityArticle.content
+          ? `<p>Could not getReadable data from the url</p>`
```

Comment:
Can we add a prop for the onErrorParse that renders that element or if you can change `getReadable` to `get reader view for ${url}`